graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -595,4 +619,14 @@ private void ensureDefaultConfiguration(String name, Collector collector) {
             LOG.error("Unable to access '{}' sidecar default configuration!", name);
         }
     }
+
+    class CollectorState {
```

Comment:
```suggestion
    static class CollectorState {
```
IntelliJ recommendation